avocado/utils/tb.py

```diff
@@ -0,0 +1,39 @@
+"""
+Traceback standard module plus some additional APIs.
```

Comment:
I understand the point made by @ruda, but also believe that @lmr motivation was to not confuse/clash it with standard library's `traceback` module.

I should have an opinion here, but I really don't.